mycroft/audio/speech.py

```diff
@@ -113,7 +114,10 @@ def mute_and_speak(utterance, ident):
         tts_hash = hash(str(config.get('tts', '')))
 
     LOG.info("Speak: " + utterance)
-    tts.execute(utterance, ident)
+    try:
+        tts.execute(utterance, ident)
```

Comment:
This eats any indication that something has failed. An except statement with an error log of the exception might be a good idea.

Reply:
There's no `except:` clause on this so any exceptions will still get raised. However, do you think we should log exceptions and suppress them here?

Reply:
I had some problems with Bing failing silently (due to expired credentials) and assumed it was here, buuut I think that was my own fault since I can see the stack trace now...

I think I was looking at the cli and and the audio log isn't picked up there. So all is good, sorry for the delay in merging :)